pkgs/development/libraries/spice-gtk/default.nix

```diff
@@ -89,16 +88,18 @@ stdenv.mkDerivation rec {
     python3.pkgs.pyparsing
     python3.pkgs.six
     vala
-    wrapGAppsHook
```

Comment:
But does it break virt-manager if you drop this?

Reply:
Removing and rebuilding yields the same result.

![20231115_20h04m53s_grim](https://github.com/NixOS/nixpkgs/assets/34864484/7bd74d86-a816-4dc6-8b31-b303d8fa062a)

Reply:
Is this virtviewer? Maybe we can add `wrapGAppsHook` to it as well.

Reply:
Actually, not sure. It's the window that open when pressing open in virt-manager.

Reply:
virt-viewer has the hook

https://github.com/NixOS/nixpkgs/blob/d4b5a67bbe9ef750bd2fdffd4cad400dd5553af8/pkgs/applications/virtualization/virt-viewer/default.nix#L32

Reply:
But maybe it's because gst is a dependency of either virt-viewer or virt-manager and thus removing the propagation breaks the package 🤔

Reply:
@Mic92 Thanks for the help! You lead me on the right track and have changed the pr to propose a fix instead.